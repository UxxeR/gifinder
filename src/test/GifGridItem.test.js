import React from "react";
import { shallow } from "enzyme";
import "@testing-library/jest-dom";
import GifGridItem from "../components/GifGridItem";

describe("GifGridItem Tests", () => {
  const wrapper = shallow(
    <GifGridItem id="gif1" title="gifaboutthings" url="" />
  );

  test("component should display correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
