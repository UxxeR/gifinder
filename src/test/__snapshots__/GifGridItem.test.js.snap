// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GifGridItem Tests component should display correctly 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img
    alt="gifaboutthings"
    src=""
  />
  <p>
    gifaboutthings
  </p>
</div>
`;
